odoo_server_maintenance/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Server Lock for Maintenance Mode',
+    'version': '8.0.1.0.0',
+    'author': 'Ecosoft,Odoo Community Association (OCA)',
+    'category': 'Hidden',
+    'description': """
```

Comment:
please remote the description key, and place it in a README file, using the template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst